redislock.go

```diff
@@ -10,25 +10,39 @@ import (
 	"github.com/redis/go-redis/v9"
 )
 
+// alias options
+var (
+	WithExpiry         = redsync.WithExpiry
+	WithDriftFactor    = redsync.WithDriftFactor
+	WithGenValueFunc   = redsync.WithGenValueFunc
+	WithRetryDelay     = redsync.WithRetryDelay
+	WithRetryDelayFunc = redsync.WithRetryDelayFunc
+	WithTimeoutFactor  = redsync.WithTimeoutFactor
+	WithTries          = redsync.WithTries
+	WithValue          = redsync.WithValue
+)
+
 // NewRedisLocker provides an implementation of the Locker interface using
 // redis for storage.
-func NewRedisLocker(r *redis.Client) (gocron.Locker, error) {
+func NewRedisLocker(r *redis.Client, options ...redsync.Option) (gocron.Locker, error) {
```

Comment:
let's make this our own type within gocron and abstract away the `redsync.Option` from the method
```suggestion
func NewRedisLocker(r *redis.Client, options ...Option) (gocron.Locker, error) {
```